app/kumactl/cmd/install/install_postgres_schema.go

```diff
@@ -0,0 +1,25 @@
+package install
+
+import (
+	"github.com/Kong/kuma/app/kumactl/pkg/install/data"
+	postgres_schema "github.com/Kong/kuma/app/kumactl/pkg/install/universal/control-plane/postgres"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+func newInstallPostgresSchemaCmd() *cobra.Command {
+	return &cobra.Command{
+		Use:   "postgres-schema",
```

Comment:
Having `Postgres` in the name of the command is opposite to removing `k8s` from `control-planes add k8s`.

E.g., `kumactl install control-plane` was deliberately designed to avoid `k8s` in the name. If at some point we add support for other platforms, we  can add `--target=k8s|universal` flag with `k8s` as a historical default value.

So, I would suggest to rename the command into `kumactl install database-schema` with implied `--target=postgres` 

Is it ok with @subnetmarco ?

Reply:
Changed, it makes sense for me.

Reply:
Yes - okay.